assets/js/filters/fees.filter.js

```diff
@@ -0,0 +1,12 @@
+angular
+  .module('walletFilters')
+  .filter('fees', feesFilter);
+
+function feesFilter () {
```

Comment:
To only display the fees we want (and to ignore the limits object inside the fees object)